pkg/backend/openshiftcluster.go

```diff
@@ -345,9 +345,16 @@ func (ocb *openShiftClusterBackend) asyncOperationResultLog(log *logrus.Entry, i
 			"properties.servicePrincipalProfile", "The Azure Red Hat Openshift resource provider service principal has been removed from your tenant. To restore, please unregister and then re-register the Azure Red Hat OpenShift resource provider.")
 	}
 
-	_, ok := backendErr.(*api.CloudError)
+	err, ok := backendErr.(*api.CloudError)
 	if ok {
-		log = log.WithField("resultType", utillog.UserErrorResultType)
+		if err.StatusCode < 300 && err.StatusCode >= 200 {
+			log.Info("long running operation succeeded")
+			return
+		} else if err.StatusCode < 500 {
+			log = log.WithField("resultType", utillog.UserErrorResultType)
+		} else {
+			log = log.WithField("resultType", utillog.ServerErrorResultType)
+		}
 	} else {
 		log = log.WithField("resultType", utillog.ServerErrorResultType)
 	}
```

Comment:
This conditional could be simplified down.

Reply:
Based on the discussion above, the `if` statement might go away entirely once we discuss and check on the only instance of `CloudError` with 3XX `StatusCode`.